src/main/java/com/debatetimer/service/customize/CustomizeService.java

```diff
@@ -18,23 +15,21 @@
 @RequiredArgsConstructor
 public class CustomizeService {
 
-    private final CustomizeTableRepository tableRepository;
-    private final CustomizeTimeBoxRepository timeBoxRepository;
+    private final CustomizeTableDomainRepository customizeTableDomainRepository;
 
     @Transactional
     public CustomizeTableResponse save(CustomizeTableCreateRequest tableCreateRequest, Member member) {
         CustomizeTable table = tableCreateRequest.toTable(member);
-        CustomizeTableEntity savedTable = tableRepository.save(new CustomizeTableEntity(table));
-
-        CustomizeTimeBoxEntities savedCustomizeTimeBoxes = saveTimeBoxes(tableCreateRequest, savedTable.toDomain());
-        return new CustomizeTableResponse(savedTable.toDomain(), savedCustomizeTimeBoxes);
+        List<CustomizeTimeBox> timeBoxes = tableCreateRequest.toTimeBoxes();
+        CustomizeTable savedTable = customizeTableDomainRepository.save(table, timeBoxes);
+        return new CustomizeTableResponse(savedTable, timeBoxes);
     }
 
     @Transactional(readOnly = true)
     public CustomizeTableResponse findTable(long tableId, Member member) {
-        CustomizeTableEntity tableEntity = tableRepository.getByIdAndMember(tableId, member);
-        CustomizeTimeBoxEntities timeBoxes = timeBoxRepository.findTableTimeBoxes(tableEntity);
-        return new CustomizeTableResponse(tableEntity.toDomain(), timeBoxes);
+        CustomizeTable table = customizeTableDomainRepository.getByIdAndMember(tableId, member);
+        List<CustomizeTimeBox> timeBoxEntities = customizeTableDomainRepository.getCustomizeTimeBoxes(tableId, member);
```

Comment:
[사소한 제안]

변수명이 timeBoxEntites인데 도메인 객체라 timeBoxes 어떨지 제안남겨보아요!
```suggestion
        List<CustomizeTimeBox> timeBoxes = customizeTableDomainRepository.getCustomizeTimeBoxes(tableId, member);
```

Reply:
수정했습니다